pyaerocom/io/readungridded.py

```diff
@@ -471,8 +628,14 @@ def _eval_filter_post(self, filter_post, data_id, vars_available):
                 filters[key] = val
         return filters
 
+    # TODO Make this method compatible with configs
```

Comment:
To be included in this PR?

Reply:
Pretty sure this works, without any major changes. But I'll leave this comment open, so I can double check that